modules/Timetable/moduleFunctions.php

```diff
@@ -2054,6 +2054,11 @@ function renderTTSpace($guid, $connection2, $gibbonSpaceID, $gibbonTTID, $title
             $output .= "<span style='font-weight: normal; font-style: italic;'>".__('Time').'<span>';
             $output .= '</th>';
             $count = 0;
+
+            // Formatter to format date as day of week (full name based on locale).
+            $dayOfWeekNameFormatter = new \IntlDateFormatter(null);
```

Comment:
I wonder if the Format class can handle this internally, so we are not using IntlDateFormatter in various places in the codebase 🤔 This way, we can possibly have a fallback case inside the Format class. What are your thoughts on Format::dayOfWeekName and Format::monthName methods, to then keep the implementation of IntlDateFormatter internally in the Format class?

Reply:
That sounds reasonable. Will work on those.

Reply:
Done.